.circleci/config.yml

```
@@ -369,13 +388,13 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: flowmachine-deps-2-{{ checksum "flowmachine/Pipfile.lock" }}
+          key: flowmachine-deps-3-{{ checksum "flowmachine/Pipfile.lock" }}
       # Need to install graphviz and pygraphviz manually because it was removed from the Pipfile
       # (see https://github.com/Flowminder/FlowKit/issues/952)
-      - run: cd flowmachine && sudo apt-get install -y libgraphviz-dev graphviz && pipenv run pip install pygraphviz
       - run: cd flowmachine && pipenv install --dev --deploy && pipenv run pip install -e .
+      - run: cd flowmachine && sudo apt-get update && sudo apt-get install -y libgraphviz-dev graphviz && pipenv install pygraphviz
```

```suggestion
      - run: cd flowmachine && sudo apt-get update && sudo apt-get install -y libgraphviz-dev graphviz && pip install pygraphviz
```

Cos otherwise it'll modify the pipfile?

---

Yeah, that's tidier